packages/vitest/src/node/watch-filter.ts

```diff
@@ -9,13 +9,18 @@ const MAX_RESULT_COUNT = 10
 const SELECTION_MAX_INDEX = 7
 const ESC = '\u001B['
 
-type FilterFunc = (keyword: string) => Promise<string[]> | string[]
+export interface FilterType {
```

Comment:
Rename this to FilterObjecr or FilterItem and replace name with key.

This way when `x` command will be added to run a test/suite we we just use key with the corresponding id.